libs/ui/src/store/wallet/wallet.epics.ts

```diff
@@ -88,11 +88,11 @@ const sendAssetEpic = (action$: Observable<Action>, state$: Observable<RootState
 
 const addNewTokensEpic = (action$: Observable<Action>) =>
   action$.pipe(
-    ofType(addNewTokensAction.submit),
+    ofType(getAllUserTokensAction.submit),
     toPayload(),
     concatMap(({ debankId, publicKeyHash }) =>
       from(getTokenList(publicKeyHash, debankId)).pipe(
-        map(tokenList => addNewTokensAction.success({ tokenList, debankGasTokenName: debankId }))
+        map(tokenList => getAllUserTokensAction.success({ tokenList, debankGasTokenName: debankId }))
```

Comment:
`debankGasTokenName: debankId` wtf?

Reply:
debankGasTokenName and debankId have the same names, and better to understand when is name debankGasTokenName in the reducer, but I can add this in reducer debankId: debankGasTokenName